asv_bench/benchmarks/packers.py

```diff
@@ -575,13 +569,19 @@ def remove(f):
         self.index = date_range('20000101', periods=self.N, freq='H')
         self.df2 = DataFrame(dict([('float{0}'.format(i), randn(self.N)) for i in range(self.C)]), index=self.index)
         self.df2['object'] = [('%08x' % randrange((16 ** 8))) for _ in range(self.N)]
-        remove(self.f)
+        self.remove(self.f)
 
     def time_packers_write_json_date_index(self):
         self.df.to_json(self.f, orient='split')
 
     def teardown(self):
-        remove(self.f)
+        self.remove(self.f)
+
+    def remove(self, f):
+        try:
+            os.remove(self.f)
```

Comment:
My expectation is that any of the benchmark modules will have a handful of classes post-cleanup. ```asv``` doesn't require different classes per benchmark, so we can put ```time_foo()``` and ```time_bar()``` on the same class. Also, these changes are all auto-generated.